### PR TITLE
Fixes #336, originally #396  Allow admins to edit users who haven't authorized any backends

### LINKED ISSUE #336: Allow admins to edit users who haven't authorized any backends
Not super high priority but should probably fix eventually.

When attempting to edit a user in admin, such as to add or remove staff status, the form will not save if the user has not authorized any backends. The `Backend metadata` field is auto-populated with an empty dictionary `{}` and is a required field, but the empty dictionary doesn't appear to be considered content and therefore is counted as an empty field, preventing saving.

Inserting the following text into the field resolves the error and allows the form to be saved, and the user is still prompted to authorize Zoom as they normally would if the field was set as `{}`
```
{
    "zoom": {}
}
```

image

As an admin, I need to be able to edit users who haven't authorized any backends without making adjustments to the contents of the `Backend metadata` field.

### PR DESCRIPTION
**Pull request by Tombow1**
Hi @ssciolla, thanks for the comments and I have generated a migration for my changes and removed the unwanted part. This is a new PR for #396 since it was originally made on the master repo.

### CODE REVIEW

File: src/officehours_api/migrations/0023_auto_20230411_0302.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.18 on 2023-04-11 07:02
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+import jsonfield.fields
+import officehours_api.models
```
Comment (ssciolla) [inline src/officehours_api/migrations/0023_auto_20230411_0302.py:6]: Can you remove any imports that are not being used?

**Review by ssciolla — COMMENTED** [review]
Thanks for opening the new PR and making the changes. I'll do some more testing on this today. Once it's approved, I think we can probably merge this before #383, and I can update that PR.

**Review by jonespm — APPROVED** [review]
This looks good to me

File: src/officehours_api/migrations/0023_auto_20230411_0302.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.18 on 2023-04-11 07:02
+from django.db import migrations, models
```
Comment (ssciolla) [inline src/officehours_api/migrations/0023_auto_20230411_0302.py:2]:
```suggestion
from django.db import migrations
```

**Review by ssciolla — APPROVED** [review]
Aside from that last unused import, this looks good! If you could remove that (which can be done in the GitHub UI), that would be great.

**Pull request merged**